docs/source/developers/overview.rst

```diff
@@ -64,12 +64,19 @@ checklist for using ``git``:
     straightforward.
 
     Once you have an updated local copy, you can push to your remote repo.  Note, since your
-    remote repo still holds the old history, you would need to do a force push.
+    remote repo still holds the old history, you would need to do a force push.  Most pushes
+    should use `--force-with-lease`
 
     .. code:: console
 
-       $ git push --force origin branch
+       $ git push --force-with-lease origin branch
+
+    If that fails, use a regular `--force` push.
```

Comment:
```suggestion
    If that fails, use a regular ``--force`` push.
```

BTW, should we try `pull` and `push --force-with-lease` before we use `--force`?

Reply:
Usually have needed to do this after rebasing. If this is done frequently, maybe better to change the git config settings to make it the default.